src/watertap_contrib/seto/multiperiod/VAGMD_batch_design_model.py

```diff
@@ -0,0 +1,386 @@
+import pandas as pd
```

Comment:
General comment/question @adam-a-a - from a repo organization standpoint, I like that @Zhuoran29 has made a separate directory for the multiperiod stuff. Presumably we will put more here. Should we have a separate directory for each multiperiod example? Any thoughts?

We have an ``analysis`` folder now with a ``net_metering`` directory, so maybe ``analysis > multiperiod > vagmd`` ? Directories on directories isn't always a good way to go so open to any suggestions

Reply:
I think it can be open for discussion. Looking at how DISPATCHES organizes its cases, it has `dispatches/case_studies/
nuclear_case/` and then all the relevant files for that case, including, but not limited to, multiperiod files. Hard to say what the best route is or if there is a best route. Whichever we choose, we should just make sure things are organized in a consistent manner (i.e., PVRO multiperiod organized same way as batch VAGMD).

Now that I think about it more, because I can imagine many potential "multiperiod" examples that can arise, it might be better to follow the way DISPATCHES did it rather than have a separate multiperiod folder.

Reply:
I put it in the main folder before because the vagmd multiperiod class function like a unit model, rather than an 'analysis', but it also makes sense to put it together with other multiperiod flowsheet.

So for now, I moved all multiperiod related files to `analysis > multiperiod > vagmd_batch`. I didn't put it in a parent folder called 'example' or 'case_studies' as DISPATCHES did, because of the reason I mentioned above that it's not really an example compared to PV_RO. We can put parallel folder there for `PV_RO` and `trough_MED`, or in a 'case_studies' folder for these applications. 

I'm not sure at this moment. I can re-organize it anytime if we see a better solution.